relayer/channel-tx.go

```diff
@@ -62,7 +62,8 @@ func (c *Chain) CreateOpenChannels(dst *Chain, maxRetries uint64, to time.Durati
 		// In the case of failure, increment the failures counter and exit if this is the 3rd failure
 		case !success:
 			failures++
-			c.Log(fmt.Sprintf("retrying transaction..."))
+			str := "retrying transaction..."
+			c.Log(str)
```

Comment:
```suggestion
			c.Log("retrying transaction...")
```
does this pass lint?

Reply:
Yes, I've done this in latest commit